Fehlermeldungen.qmd

```diff
@@ -371,7 +371,7 @@ Wenn man im ersten Durchlauf nichts gefunden hat, ist es beispielsweise sinnvoll
 
 Manchmal ist ein Fehler so global, dass er bei verschiedenen Funktionen auftaucht. Daher lohnt es sich manchmal auch, eine __Suche *ohne* den Funktionsnamen__ durchzuführen. Spätestens, wenn man bei der Suche mit allen drei Suchbegriffen nichts gefunden hat, ist es sinnvoll, nur mit **Programm** + **Fehlermeldung** zu suchen.
 
-Wir können auch auf **allgemeine <a href="https://www.lifewire.com/google-searches-and-power-tricks-4093009" target="_blank">Tipps zum Suchen im Internet</a>** zurückgreifen, u.a.:
+Wir können auch auf allgemeine **<a href="https://zapier.com/blog/advanced-google-search-tricks/" target="_blank">Tipps zum Suchen im Internet</a>** zurückgreifen, u.a.:
```

Comment:
Ich würde hier eher auf Tricks wie diese hier verweisen: https://www.theguardian.com/technology/2016/jan/15/how-to-use-search-like-a-pro-10-tips-and-tricks-for-google-and-beyond